aioaws/s3.py

```diff
@@ -29,9 +29,10 @@
 @dataclass
 class S3Config:
     aws_access_key: str
-    aws_secret_key: str
+    aws_secret_key: str = field(repr=False)
     aws_region: str
     aws_s3_bucket: str
+    aws_session_token: str = field(repr=False, default='')
```

Comment:
again, is there a reason to use `''` instead of `None` as default?

Reply:
Again, I used an empty string as a default to keep the type signature consistent with `S3ConfigProtocol.aws_session_token`. Otherwise the type signature would be `Optional[str]`.

A default of `None` could also impact use of `_utils.get_config_attr`, because this method expects all config attributes to be strings.

https://github.com/samuelcolvin/aioaws/blob/9e938435da88bb0be26b21b834f37e6cd759e15e/aioaws/_utils.py#L25